stack.yaml

```
@@ -1,12 +1,12 @@
-resolver: lts-12.10
-compiler: ghc-8.4.4
+resolver: lts-14.12
 
 packages:
 - .
 
 extra-deps:
 - network-uri-static-0.1.1.0
-
+- stm-containers-0.2.16
+- focus-0.1.5.2
```

```suggestion
- focus-0.1.5.2  # this is the last version that works with stm-containers-0.2.16
```

actually, this also seems to work:

```yaml
extra-deps:
- network-uri-static-0.1.1.0
- stm-containers-1.1.0.4
- stm-hamt-1.2.0.4
```

but requires some changes in the hscim code to accommodate for the changes in those libs.

so i'm in favor of leaving the extra-deps as is for now.